src/app/services/solver/solver.service.ts

```diff
@@ -7,9 +7,13 @@ import {Node} from '../../gte-core/gte/src/Model/Node';
   providedIn: 'root'
 })
 export class SolverService {
-  private url = `http://test.api.logos.bg/api/solve/`;
+  private url = `http://localhost:8000/api/solve/`;
```

Comment:
this will not work unless on localhost.

Reply:
Can we use an environment variable (e.g. set in the Docker Compose file)? This would make it easier to deploy GTE to other servers or run it locally.